server/server.py

```diff
@@ -0,0 +1,115 @@
+import os
+
+from flask import Flask, request
+from lib.broadlink_controller import BroadlinkController
+from lib.custom_device import CustomDevice
+
+app = Flask(__name__)
+
+controllers = dict()
+custom_devices = dict()
+
+
+@app.route('/send_command', methods=['GET'])
+def send_command():
+    command = request.args.get('command')
+    custom_device_name = f"{request.args.get('custom_device_name')}.device"
+
+    custom_device_list = custom_devices.keys()
+
+    if custom_device_name not in custom_device_list:
+        return f"custom device '{request.args.get('custom_device_name')}' not found", 404
+
+    selected_custom_device: CustomDevice = custom_devices.get(custom_device_name)
+
+    if command and command in selected_custom_device.get_commands():
+        selected_custom_device.send_command(command)
+        return "Command sent successfully", 200
+    elif command:
+        return f"Invalid command: {command}", 400
+    else:
+        return "Missing 'command' parameter", 400
+
+
+@app.route("/register_controller", methods=["GET"])
```

Comment:
I am thinking to just pull this registration out into a config file and/or to generate the config file from a "custom device creator cli"

Reply:
ugh sorry, i saw this just now, you know what, i'm gonna transfer this fork to you, i have literally zero free time to spend on this and you seem to be eager to develp this further, you shouldnt be waiting for me.